tests/yam/validate/validate_desktop.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+
+# Summary: Validate agama GNOME Desktop Environment (Wayland)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "x11test";
+use Utils::Architectures qw(is_s390x is_ppc64le);
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    return if (is_s390x() || is_ppc64le());
```

Comment:
we will work in the future with opting in/out of validation, but for now the normal way is to create different schedules. I guess in remote worker you start the system installed in multi-user so no desktop running, but still we should check the pattern installed that should be common.

Reply:
OK, will create different schedule for these 2 arches. Thanks